src/CMakeLists.txt

```diff
@@ -1,8 +1,12 @@
 add_library(libultraship STATIC)
 set_target_properties(libultraship PROPERTIES PREFIX "")
 set_property(TARGET libultraship PROPERTY CXX_STANDARD 20)
+# Need to set C11 for libgfxd (_Alignas)
+set_property(TARGET libultraship PROPERTY C_STANDARD 11)
```

Comment:
*newer* versions don't work?

Reply:
I don't think it won't work, but the default didn't.

Reply:
I removed this line and didn't have issues building, so it could probably be dropped. `alignas` has been in c standard since C11.